independent-publisher-connectors/Spotify/apiDefinition.swagger.json

```diff
@@ -0,0 +1,943 @@
+                },
+                "previous": {
+                  "type": "string",
+                  "x-ms-summary": "Previous",
+                  "description": "URL to the previous page of items. ( null if none)"
+                },
+                "total": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Total",
+                  "description": "The total number of items available to return."
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get User's Saved Albums",
+        "description": "Get a list of the albums saved in the current Spotify user's 'Your Music' library.",
+        "operationId": "GetSavedAlbums",
+        "parameters": []
+      }
+    },
+    "/v1/me": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "$ref": "#/definitions/PrivateUser"
+            }
+          }
+        },
+        "summary": "Get Current User's Profile",
+        "description": "Get detailed profile information about the current user (including the current user's username).",
+        "operationId": "GetUserProfile",
+        "parameters": []
+      }
+    },
+    "/v1/me/top/artists": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "items": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/Artist"
+                  },
+                  "description": "The current user's top artists."
+                },
+                "href": {
+                  "type": "string",
+                  "x-ms-summary": "Href",
+                  "description": "A link to the Web API endpoint returning the full result of the request"
+                },
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Limit",
+                  "description": "The maximum number of items in the response (as set in the query or by default)."
+                },
+                "next": {
+                  "type": "string",
+                  "x-ms-summary": "Next",
+                  "description": "URL to the next page of items. ( null if none)"
+                },
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "x-ms-summary": "Offset",
+                  "description": "The offset of the items returned (as set in the query or by default)"
+                },
+                "previous": {
+                  "type": "string",
+                  "x-ms-summary": "Previous",
+                  "description": "URL to the previous page of items. ( null if none)"
```

Comment:
@Laskewitz Can you please remove the extra space in the rounded brackets.

`"description": "URL to the next page of items. (null if none)"`

Reply:
Fixed.